### ISSUE TITLE
TypeError: expected string or buffer

### ISSUE THREAD

**Issue by elieobeid7** [Diaoul/subliminal#771]
When there's a movie, then a folder, then a movie, then a subfolder, then a movie I get an error. but if there's no subfolders, the code works perfectly. My code is
```

if (subtitlesBoolean == True):
    region.configure('dogpile.cache.dbm', arguments={
                     'filename': 'cachefile.dbm'})
    videos = scan_videos(inputPath)
    subtitles = download_best_subtitles(videos, {Language('eng')})
    for video in videos:
        save_subtitles(video, subtitles[video])
```
The error
```
  File "main.py", line 82, in <module>
    subtitles = download_best_subtitles(videos, {Language('eng')})
  File "/home/elie/.virtualenvs/tv/lib/python2.7/site-packages/subliminal/core.py", line 647, in download_best_subtitles
    compute_score=compute_score)
  File "/home/elie/.virtualenvs/tv/lib/python2.7/site-packages/subliminal/core.py", line 211, in download_best_subtitles
    for s in subtitles], key=operator.itemgetter(1), reverse=True)
  File "/home/elie/.virtualenvs/tv/lib/python2.7/site-packages/subliminal/score.py", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/home/elie/.virtualenvs/tv/lib/python2.7/site-packages/subliminal/providers/podnapisi.py", line 52, in get_matches
    if video.series and sanitize(self.title) == sanitize(video.series):
  File "/home/elie/.virtualenvs/tv/lib/python2.7/site-packages/subliminal/utils.py", line 111, in sanitize
    string = re.sub(r'[%s]' % re.escape(''.join(characters)), ' ', string)
  File "/home/elie/.virtualenvs/tv/lib64/python2.7/re.py", line 155, in sub
    return _compile(pattern, flags).sub(repl, string, count)
TypeError: expected string or buffer
```

**Comment by thomasloldk** [comment]
#775  - Same bug/error

**Comment by elieobeid7** [comment]
whenever it's fixed please let me know, thanks for the notification

**Comment by peppot** [comment]
Still happening here. Can't use subliminal with automation in a large dir

**Comment by ratoaq2** [comment]
Is this still an issue?
More info needed here. 
Does it work with python 3?
What's the OS/file system encoding?
What's the problematic folder name?

If someone could provide me a way to reproduce it, I could fix it

**Comment by ratoaq2** [comment]
Just found out is the same as #775 and it was fixed by #761
